pkg/datapath/linux/requirements.go

```diff
@@ -36,9 +36,18 @@ func CheckRequirements(log *slog.Logger) error {
 		}
 	}
 
-	// bpftool checks
 	if !option.Config.DryMode {
-		probeManager := probes.NewProbeManager(log)
+		if err := probes.HaveBPF(); err != nil {
+			return errors.New("Require support for bpf() (CONFIG_BPF_SYSCALL=y)")
+		}
+
+		if err := probes.HaveBPFJIT(); err != nil {
+			return errors.New("Require support for the eBPF JIT (CONFIG_HAVE_EBPF_JIT=y and CONFIG_BPF_JIT=y)")
+		}
+
+		if err := probes.HaveIngressClassifier(); err != nil {
+			return errors.New("Require support for the clsact qdisc (CONFIG_NET_CLS_ACT=y) and ingress classes (CONFIG_NET_SCH_INGRESS=y)")
```

Comment:
This should not be a hard requirement. We require either this one or tcx. The kernel could also have tcx with legacy CONFIG_NET_CLS_ACT compiled out completely.